src/main/asciidoc/java/index.adoc

```diff
@@ -0,0 +1,132 @@
+= Vert.x MySQL / PostgreSQL service
+
+The `link:apidocs/io/vertx/ext/asyncsql/AsyncSqlService.html[MySQL / PostgreSQL Service]` is responsible for providing an
+interface for Vert.x applications that need to interact with a MySQL or PostgreSQL database.
+
+It uses Mauricio Linhares https://github.com/mauricio/postgresql-async[open source driver] to interact with the MySQL
+or PostgreSQL databases in a non blocking way
+
+== Setting up the service
+
+As with other services you can use the service either by deploying it as a verticle somewhere on your network and
+interacting with it over the event bus, either directly by sending messages, or using a service proxy, e.g.
+
+Somewhere you deploy it:
+
+[source,java]
+----
+JsonObject config = new JsonObject().put("host", "mymysqldb.mycompany");
+DeploymentOptions options = new DeploymentOptions().setConfig(config);
+
+// Service name is "io.vertx:postgresql-service" if you want postgreSql
```

Comment:
Yes, there are now two services. I think it's cleaner that way and I didn't like the way the previous config contained a mysql: or postgresl: block.